<launch>
  <arg name="timeout" default="1.0" />
  <arg name="threshold" default="1.57" />
  <arg name="shutdown" default="false" />
  <arg name="component_name" default="base" />
  
  <group ns="$(arg component_name)">
    <node pkg="cob_base_controller_utils" type="cob_stuck_detector" name="stuck_detector">
        <param name="timeout" type="double" value="$(arg timeout)"/>
        <param name="threshold" type="double" value="$(arg threshold)"/>
        <param name="shutdown" type="bool" value="$(arg shutdown)"/>
    </node>
  </group>
</launch>
